src/App.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import {NavigationContainer} from '@react-navigation/native';
```

Comment:
Another TODO comment: https://reactnavigation.org/docs/handling-safe-area/
iOS looks funky when launched.
![simulator_screenshot_3E369113-7829-4B11-B957-4DA16A1EB07B](https://user-images.githubusercontent.com/39040273/208920666-aabf83b8-349d-48d3-9ade-2b036493bc5f.png)

Reply:
Addressed in #14